contributions/presentation/weak2/isacarv-anhenri/README.md

```diff
@@ -0,0 +1,17 @@
+# Presentation Proposal
+
+ ## Members
+ Isac Arvidsson (isacarv@kth.se)
+ Github username: [isacarvid](https://github.com/isacarvid)
+
+ Andreas Henriksson (anhenri@kth.se)
+ Github username: [heeenkie](https://github.com/heeenkie)
+
+ ## Proposal
+ Docker vs Kubernetes
+
+-introduction to containerization.
+-A brief summary of the two tools.
+-Differens between the tools
```

Comment:
It would be better to focus on one specific aspect of one of the tools.

Reply:
So would just a presentation of the docker tool do it?

-information about containerization
-the docker tool
-usecases

Reply:
@heeenkie a presentation about the basics of Docker is not enough. 
Docker is not new anymore. You need to focus your presentation on a specific Docker user case in detail.